src/Modules/Shared/PowerShellGet/PSModule.psm1

```diff
@@ -13974,7 +13974,7 @@ function ValidateAndGet-AuthenticodeSignature
             # Skip the PSGetModuleInfo.xml and ModuleName.cat files in the catalog validation
             $TestFileCatalogResult = Microsoft.PowerShell.Security\Test-FileCatalog -Path $ModuleBasePath `
                                                                                     -CatalogFilePath $CatalogFilePath `
-                                                                                    -FilesToSkip $script:PSGetItemInfoFileName,'*.cat' `
+                                                                                    -FilesToSkip $script:PSGetItemInfoFileName,'*.cat','*.nupkg','*.nuspec' `

```

Comment:
Is this line part of the workaround?  If not, maybe it should be in another commit.